dts/bindings/npmx/nordic,npm1300_eng_b.yaml

```diff
@@ -0,0 +1,34 @@
+# Copyright (c) 2022 Nordic Semiconductor ASA
+# SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+
+description: |
+    This is a representation of the nPM1300 PMIC device
+
+compatible: "nordic,npm1300_eng_b"
+
+include: i2c-device.yaml
+
+properties:
+  host-int-gpios:
+    type: phandle-array
+    required: true
+    description: |
+      Host GPIO pin used as interrupt input.
+  pmic-int-pin:
+    type: int
+    required: true
+    description: |
+      nPM GPIO pin number used as interrupt output.
+  host-pof-gpios:
+    type: phandle-array
+    description: |
+      Host GPIO pin used as power loss warning input.
+  pmic-pof-pin:
+    type: int
+    description: |
+      nPM GPIO pin number used as power loss warning output.
+  pmic-reset-pin:
+    type: int
+    description: |
+      nPM GPIO pin number used as watchdog warning reset output.
```

Comment:
This value is only stored in the driver config structure. What is the point of providing such property?

Reply:
`pmic-reset-pin` is used in samples where the reset pin is required. It is a hardware connection between pmic and soc reset, so we decided to keep it in devicetree.

Also if `pmic-int-pin` is configured in overlay, `pmic-reset-pin` also should be, for example:
![image](https://github.com/nrfconnect/sdk-nrf/assets/108533244/7a2470af-dd23-43fc-bbcf-10c5d0dfddd1)

Where gpio 0 is used as an interrupt pin and gpio 1 is used as reset pin.